ROADMAP.md

```diff
@@ -259,107 +265,187 @@ Any dates listed below and the specific issues that will ship in a given milesto
+    * `Composite Resource Claims` (XRCs) replace Requirements and they no longer require any specific kind suffix.
+  * Hardening and robustness enhancements towards v1beta1 quality
 
-## v0.13
-* Infrastructure Composition
-  * Progress towards v1beta1 quality
-  * Select random Composition when multiple match
+* Package Manager
+  * Streamlined v2 design
+    [crossplane#1616](https://github.com/crossplane/crossplane/pull/1616)
+    * Supports installing and managing Crossplane `Providers` and `Configurations`
+  * Package Manager v2
+    [crossplane#1675](https://github.com/crossplane/crossplane/pull/1675)
+    * Upgrade and rollback support
+    * Faster package deploys
+    * Paves the way for automatic package dependency resolution
+
+* RBAC Manager
+  * Automatically manages the RBAC roles and bindings required by `Providers` and `Composite` resources
+  * An optional deployment that uses RBAC privilege escalation
+  * Crossplane no longer requires cluster-admin privileges.
+
+* Providers
+  * General
+    * Default `ProviderConfig` supported & migration
+    * Removed deprecated claims/classes - you can now create your own claim kinds with Composition
+
+  * AWS Provider: more API types [provider-aws#149](https://github.com/crossplane/provider-aws/issues/149)
+    * S3 Bucket to v1beta1 [#331](https://github.com/crossplane/provider-aws/pull/331)
+    * S3 Bucket Policy support [#289](https://github.com/crossplane/provider-aws/pull/289)
+    * Referencer for SubnetGroup AWS ElasticCache [#314](https://github.com/crossplane/provider-aws/pull/314)
+    * Add ARN to AtProvider for SNS Topic [#348](https://github.com/crossplane/provider-aws/pull/348)
+    * ECR support [#307](https://github.com/crossplane/provider-aws/issues/307)
+
+  * Helm Provider
+    * experimental support - for use in `Compositions`
+
+  * Code generation of Crossplane providers
+    * Evaluate generating native Crossplane providers with existing code gen pipelines
+    * Evaluate wrapping stateless Terraform providers (work-in-progress) [#262](https://github.com/crossplane/crossplane/issues/262)
+
+* Open Application Model (OAM)
+  * Moved AppConfig controller out of core
+    * Install via: `helm install crossplane` with the `--set alpha.oam.enabled=true` flag
+  * Enhance health scope with informative health condition [#194](https://github.com/crossplane/oam-kubernetes-runtime#194)
+  * Add component webhook to support workload definition type [#198](https://github.com/crossplane/oam-kubernetes-runtime#198)
+  * Add health check support for containerized.standard.oam.dev in Health [#214](https://github.com/crossplane/oam-kubernetes-runtime#214)
+  * Run with fewer privileges [#228](https://github.com/crossplane/oam-kubernetes-runtime/pull/228)
+  * Hardening and robustness enhancements towards v1beta1 quality
+
+## v0.14 - v1beta1 APIs in core, hardening, robustness (short release)
```

Comment:
i wonder if next time we should consider reverse sorting this list, so the newest milestones are on top

Reply:
agree, thinking the same.

Reply:
or maybe a toc so you can just jump, so you're not looking at the longer term roadmap the very first thing

Reply:
will add a toc, just a second

Reply:
@jbw976 added toc and reordered, how does that look?